readme.markdown

```diff
@@ -1024,7 +1024,7 @@ Use dot-separated numbers to represent the release version in the git
 tag label. Nimble will parse these git tag labels to know which
 versions of a package are published.
 
-``` text
+```sh
```

Comment:
```suggestion
``` text
```

Reply:
text will not show comments

Reply:
Well it's not shell code so I guess don't use any language tag then.